components/table/TableRowExpanded.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Table } from '@oacore/design'
+import classNames from 'classnames'
+
+import tableClassNames from './index.css'
+
+const TableRowExpanded = ({
+  id,
+  isExpanded,
+  selectable,
+  config: { expandedRowRenderer },
+  ...props
+}) => {
+  return (
+    <Table.Row
+      id={`${id}-expanded`}
+      className={classNames(tableClassNames.additionalRow, {
+        [tableClassNames.closed]: !isExpanded,
+      })}
+      aria-hidden={!isExpanded}
+    >
+      {selectable && <Table.Cell />}
+      <Table.Cell colSpan={1000}>{expandedRowRenderer(props)}</Table.Cell>
```

Comment:
It might be good to try rendering this cell in the same row but hidden and then display it for expanded state if you did not do so.

Reply:
I didn't think of it like that. Not sure if it would work because of animations. For example, oai label should expand on click. This would recreate that element and no animation would be applied.

Reply:
It should not recreate, only class should change. Although, you have a point.